source/features/sort-issues-by-update-time.js

```diff
@@ -0,0 +1,44 @@
+import select from 'select-dom';
+import {anySelector} from '../libs/utils';
+
+export default function () {
+	// Get issues links that don't already have a specific sorting applied
+	const links = select.all(anySelector(`
+		:any(
+			[href*="/issues"],
+			[href*="/pulls"]
+		):not([href*="sort%3A"]):not(.issues-reset-query)
+	`));
+	for (const link of links) {
+		// Pick only links to lists, not single issues
+		if (!/(issues|pulls)\/?$/.test(link.pathname)) {
```

Comment:
What does this check for exactly? `issue` and `pull` aren't matched by `:any([href*="/issues"], [href*="/pulls"])`